src/main/java/org/sigmah/server/servlet/exporter/template/IndicatorEntryCalcTemplate.java

```diff
@@ -218,6 +218,7 @@ private void createDetailSheet(final IndicatorDTO indicator) throws Throwable {
 		Map<String, Integer> columnIndexMap = new HashMap<String, Integer>();
 		for (PivotTableData.Axis axis : leaves) {
 			CalcUtils.putHeader(row, ++cellIndex, axis.getLabel());
+            axis.setLabel(String.valueOf(cellIndex));
```

Comment:
@manishvishnoi2 : This needs an indentation fix before the pull request can be merged.